okexchain/chain.json

```diff
@@ -0,0 +1,108 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "exchain",
```

Comment:
chain name in both files must match folder name. Folder: okexchain, assetlist.json: okxchain, chain.json: exchain. Please pick just one to use in all 3 places.

Reply:
imo, 'exchain' would be harder to find in an alphabetized list than something starting with 'ok...', so I'd recommend either 'okxchain' or 'okexchain'

Reply:
Got it ,Thanks